docs/Dialects/SV/RationaleSV.md

```diff
@@ -141,6 +141,21 @@ sv.verbatim "MACRO({{0}}, {{1}} reg={{4}}, {{3}})"
             {symRefs = [@reg1, @Module1, @instance1]}
 ```
 
+Substitions also allow format specifier after a ':'. The meaning of said options
+depends on the of the operand or symbol. So far, the follow format specifiers
```

Comment:
Looks like missing a word, maybe? "on the of the operand or symbol" -- not quite sure.

Also:
```suggestion
depends on the of the operand or symbol. So far, the following format specifiers
```